fcrepo-client-impl/src/main/java/org/fcrepo/client/impl/FedoraResourceImpl.java

```diff
@@ -97,15 +97,118 @@ public FedoraResourceImpl(final FedoraRepository repository, final HttpHelper ht
     }
 
     @Override
-    public void copy(final String destination) throws ReadOnlyException {
-        // TODO Auto-generated method stub
-        throw new NotImplemented("Method copy(final String destination) is not implemented.");
+    public void copy(final String destination) throws FedoraException {
```

Comment:
Please create a new integration test minimally for your new operations: `copy`, `delete`, `move` (and ideally for the other public methods on this class).
This integration test will be very similar to `FedoraRepositoryImplIT`.